src/Microsoft.IdentityModel.Tokens/CryptoProviderCache.cs

```diff
@@ -37,7 +37,7 @@ public abstract class CryptoProviderCache
         /// </summary>
         /// <param name="signatureProvider">the <see cref="SignatureProvider"/> to create the key for.</param>
         /// <returns>the cache key to use for finding a <see cref="SignatureProvider"/>.</returns>
-        protected abstract string GetCacheKey(SignatureProvider signatureProvider);
+        protected internal abstract string GetCacheKey(SignatureProvider signatureProvider);
```

Comment:
This is breaking.

Reply:
@brentschmaltz, how can this be breaking? According MS Doc: "A protected internal member is accessible from the current assembly or from types that are derived from the containing class.". I made the change so we don't need to create the CryptoProviderCacheTests.InMemoryCryptoProviderCachePublic class in order to test the protected methods. If this is not a desired practice, I can change it back.